test/web-platform-tests/start-wpt-server.js

```diff
@@ -66,8 +67,12 @@ module.exports = ({ toUpstream = false } = {}) => {
 };
 
 function pollForServer(url, lastLogTime = Date.now()) {
-  return requestHead(url, { strictSSL: false })
-    .then(() => {
+  const agent = url.startsWith("https") ? new https.Agent({ rejectUnauthorized: false }) : null;
```

Comment:
I'm surprised this is necessary in the testing context. Have you tried without?

Reply:
I have tried leaving out the "agent" option in the fetch call, which caused the before-hook in Mocha to crash. Basically, without it we cannot poll the https address.